core/controllers/editor.py

```diff
@@ -701,9 +701,15 @@ def get(self, exploration_id, exploration_version):
         except:
             raise self.PageNotFoundException
 
-        self.render_json(stats_services_old.get_exploration_stats(
-            current_exploration.id, exploration_version,
-            current_exploration.states))
+        if feconf.ENABLE_NEW_STATS_FRAMEWORK:
+            if exploration_version == 'all':
```

Comment:
Use stats_services.VERSION_ALL

Reply:
Not required anymore since we are not passing version